starter-code/src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-function Soldier () {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health -= damage;
+};
+
 
 // Viking
-function Viking () {}
+Viking.prototype = Object.create(Soldier.prototype);
+
+function Viking(name, health, strength) {
+  this.name = name;
+  this.health = health;
```

Comment:
tanto el health como la strenth podrías heredarlas con Soldier.call(this, health, strength)